time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -49,41 +49,21 @@ def create_sql_ignore_id_condition(id: str):
         else:
             return "AND c.id!=@ignore_id"
 
-    @staticmethod
-    def create_sql_date_range_filter(date_range: dict) -> str:
-        if 'start_date' and 'end_date' in date_range:
-            return """
-            ((c.start_date BETWEEN @start_date AND @end_date) OR
-             (c.end_date BETWEEN @start_date AND @end_date))
-            """
-        else:
-            return ''
-
     def find_all_entries(
         self,
         event_context: EventContext,
         conditions: dict = None,
-        custom_sql_conditions: List[str] = None,
         date_range: dict = None,
         **kwargs,
     ):
         conditions = conditions if conditions else {}
-        custom_sql_conditions = (
-            custom_sql_conditions if custom_sql_conditions else []
-        )
         date_range = date_range if date_range else {}
 
-        custom_sql_conditions.append(
-            self.create_sql_date_range_filter(date_range)
-        )
-
-        custom_params = self.generate_params(date_range)
         time_entries = CosmosDBRepository.find_all(
```

Comment:
We may consider using query builder here also.

Reply:
Ok, adjustments are done.